### PR TITLE
Updated checkbox logic, enabled jQuery

### PR DESCRIPTION
**Pull request by smwatts**
Updated checkbox logic using jQuery, added functionality to our app where we can now use jQuery in the app-jquery.js file. jquery.js supplies this functionality. Will update static files on the server after PR is approved :)

### CODE REVIEW

File: taworks/taform/static/taform/js/app-jquery.js [hunk 1]
```diff
@@ -0,0 +1,14 @@
+$(document).ready(function() {
+
```
Comment (j2kan) [inline taworks/taform/static/taform/js/app-jquery.js]: remove space here and below

**Review by j2kan — DISMISSED** [review]
holy shit you're adding +10,387 lines of code.

thanks sarah lgtm :D

**Review by lauradobson — CHANGES_REQUESTED** [review]
Tested.

I checked all, then hit "I agree", then unchecked one box and I was still able to submit the application.

**Comment by smwatts** [comment]
Did you run it from the checkbox branch? And run python manage.py collectstatics first?
The "I agree" button shouldn't be there

screen shot 2018-01-25 at 10 14 23 am

**Review by amyleblond — APPROVED** [review]
good!

**Review by lauradobson — APPROVED** [review]

**Pull request merged**